internal/gps/lock.go

```diff
@@ -195,6 +196,15 @@ func (lp LockedProject) Packages() []string {
 	return lp.pkgs
 }
 
+func (lp LockedProject) String() string {
+	prj := lp.Ident()
+	rev, _, _ := VersionComponentStrings(lp.Version())
```

Comment:
hmm...i think that only showing the revision here is probably not the optimal UX. it'd be better to show the full paired information - version or branch, and then rev in parens. it can follow the same pattern as in `GetLockingFeedback()`. the logic will have to be duplicated, though - not only do we try very hard to encapsulate gps into having no knowledge of dep, but that would create an import cycle.